modules/openthread/Kconfig.thread

```diff
@@ -168,6 +168,7 @@ config OPENTHREAD_MAC_STAY_AWAKE_BETWEEN_FRAGMENTS
 config OPENTHREAD_DEFAULT_TX_POWER
 	int "OpenThread default tx power in dBm"
 	range -40 20 if NRF_802154_RADIO_DRIVER
+	range -30 9 if BT_B91
```

Comment:
I don't think that bluetooth has relation to openthread...

Reply:
I know. I couldn't find any more proper define for this one. I hope someone has better idea ;) or maybe we should add sth respectively to NRF?

Reply:
replaced with IEEE802154_TELINK_B91